Source/UINavigation/Public/UINavMacros.h

```diff
@@ -7,4 +7,4 @@
 #define DISPLAYERROR(Text) GEngine->AddOnScreenDebugMessage(-1, 10.f, FColor::Red, FString::Printf(TEXT("%s"), *(FString(TEXT("Error in ")).Append(GetName()).Append(TEXT(": ")).Append(Text))))
 #define DISPLAYERROR_STATIC(Widget, Text) GEngine->AddOnScreenDebugMessage(-1, 10.f, FColor::Red, FString::Printf(TEXT("%s"), *(FString(TEXT("Error in ")).Append(Widget->GetName()).Append(TEXT(": ")).Append(Text))))
 #define DISPLAYWARNING(Text) GEngine->AddOnScreenDebugMessage(-1, 10.f, FColor::Orange, FString::Printf(TEXT("%s"), *(FString(TEXT("Warning in ")).Append(GetName()).Append(TEXT(": ")).Append(Text))))
-#define IS_VR_PLATFORM !PLATFORM_SWITCH && !PLATFORM_XBOXONE
```

Comment:
Is there no more XBOX_ONE or did it change name?

Reply:
IDK, the compiler complains about the Xbox One but not the Switch...